widgets/docs/getting_started.md

```diff
@@ -4,16 +4,17 @@
 
 To install the widget, do the following:
 
-1. `npm i avatar-widgets`
+1. `npm i persona-link-avatar --force`
 2. Before using the avatar-widget please make sure that your backend example server must be serving on 8000 port
-3. After that import the widget into your React component: `import Avatarwidget from 'avatar-widgets';`
+3. After that import the widget into your React component: `import Avatarwidget from 'persona-link-avatar';`
 4. Then, you can use it in your component's render method like:
 
 ```
-render() {
+function name() {
   return (
 
-      <Avatarwidget message="nice to meet you mr. john" />
+     
+      {conversationId&&( <Avatarwidget conversationid= {conversationId} websocketadd="localhost:9000" />)}
```

Comment:
make sure you mention that conversationId must be part of state of the component.